Packs/SpyCloudEnterpriseProtection/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "SpyCloud Enterprise Protection",
     "description": "Create breach and malware incidents in Cortex® XSOAR™ using the SpyCloud Enterprise Protection API. Provide enrichment for domains, IPs, emails, usernames, and passwords.",
     "support": "partner",
-    "currentVersion": "1.0.7",
+    "currentVersion": "1.1.0",
```

Comment:
it's a small change, so please change it to be 1.0.8 instead, and change the release notes file name as well